client/src/components/HeroImg.jsx

```diff
@@ -1,10 +1,19 @@
 import React from "react";
+import imgs from "../static/static-imgs/jacqueline-munguia-1pAwJiCD60c-unsplash.jpg";
 
 export const HeroImg = () => {
   return (
-    <div className="container col s6" id="heroImg">
+    <div className="container" id="heroImg">
       <div className="row">
-        <div className="">HeroImg</div>
+        <div className="">
+          {" "}
+          <img
+            className="materialboxed"
+            height="650"
+            width="650"
+            src={"/static-imgs/jacqueline-munguia-1pAwJiCD60c-unsplash.jpg"}
```

Comment:
```suggestion
            src={imgs}
```
If the `import imgs` above is successful then the above change should work

Reply:
@jaygraygray that worked. Ty!